src/App.tsx

```diff
@@ -29,7 +29,7 @@ function App() {
     const [url, setUrl] = useState<string>('');
     const [playlistItems, setPlaylistItems] = useState<PlaylistedTrack<Track>[] | null>(null);
     const [name, setName] = useState<string>('');
-    const [codeType, setCodeType] = useState('qr');
```

Comment:
Could you just add a select dropdown to the frontend to change this behaviour, then this would finally do the thing I did not come around doing since the last time i worked on this...
The rest looks good an can be merged!

Reply:
Ah. I see that the other option would be to create spotify codes. As mentioned, I actually do not know typescript or nodejs. I know even less about frontend and react. But I'll see if ChatGPT makes it happen :D

Reply:
Done